src/components/Dashboard/FilingHistoryList/FutureEffective.vue

```diff
@@ -22,50 +22,51 @@
 </template>
 
 <script lang="ts">
-import { Component, Vue, Prop } from 'vue-property-decorator'
-import { mapState } from 'vuex'
+import { Component, Mixins, Prop } from 'vue-property-decorator'
+import { Getter } from 'vuex-class'
+import { DateMixin } from '@/mixins'
 import { ContactInfo } from '@/components/common'
 import { HistoryItemIF } from '@/interfaces'
 
 @Component({
-  computed: { ...mapState(['entityName']) },
   components: { ContactInfo }
 })
-export default class FutureEffective extends Vue {
-  readonly entityName!: string
+export default class FutureEffective extends Mixins(DateMixin) {
+  @Getter getEntityName!: string
 
   /** The subject filing. */
-  @Prop({ required: true }) private filing: HistoryItemIF
+  @Prop({ required: true })
+  readonly filing: HistoryItemIF
 
   /** Data for the subject filing. */
-  private get _ (): any {
+  get _ (): any {
```

Comment:
I'm unfamiliar with the syntax here, could you explain for my know-how :)

Reply:
There's nothing fancy with the syntax. I thought I'd copy the Lodash example of importing their methods into a class named `_` (eg, _.cloneDeep). So this getter is named `_`.

The idea is to have something simple, that isn't `this`. I could have named it `data` but that seemed weird and repetitive instead of clean.